main.go

```diff
@@ -60,6 +61,15 @@ func getBundlerVersion() (gems.Version, error) {
 	return gems.ParseBundlerVersion(lockFileContent)
 }
 
+func shellQuote(params string) ([]string, error) {
```

Comment:
this function is equal to `args, err := shellquote.Split(params)` please remove the unnecessary wrapper fn